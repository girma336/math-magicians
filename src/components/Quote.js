import './styles/Quote.css';

const Quote = () => (
  <div className="quout-gen">
    <p>
      &quot; Never do things others can do
      and will do, if there are things others cannot do or will not do &quot;  -
      <span className="author"> Amelia Earhart</span>
    </p>
  </div>
);

export default Quote;
