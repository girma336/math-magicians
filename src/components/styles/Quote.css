.quout-gen {
  width: 95%;
  text-align: center;
  border: 1px solid #666;
  box-shadow: 0 0 5px #666;
  border-radius: 8px;
  margin: 0 auto;
}

p {
  font-size: 18px;
}
