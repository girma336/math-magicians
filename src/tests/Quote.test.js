import renderer from 'react-test-renderer';
import Quote from '../components/Quote';

describe('Test by Using Snapshot ofr Quote', () => {
  it('rendere curectlly for quote component', () => {
    const tree = renderer.create(<Quote />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
