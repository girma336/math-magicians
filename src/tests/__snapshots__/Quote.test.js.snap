// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test by Using Snapshot ofr Quote rendere curectlly for quote component 1`] = `
<div
  className="quout-gen"
>
  <p>
    " Never do things others can do and will do, if there are things others cannot do or will not do "  -
    <span
      className="author"
    >
       Amelia Earhart
    </span>
  </p>
</div>
`;
